src/WebsockettestDatastructures.js

```diff
@@ -153,14 +153,14 @@ function RMBTTestResult() {
 RMBTTestResult.prototype.addThread = function(rmbtThreadTestResult) {
     this.threads.push(rmbtThreadTestResult);
 };
-RMBTTestResult.prototype.ip_local;
-RMBTTestResult.prototype.ip_server;
-RMBTTestResult.prototype.port_remote;
-RMBTTestResult.prototype.num_threads;
+RMBTTestResult.prototype.ip_local = undefined;
```

Comment:
Which IDE are you using? But yes, please change it to null instead of undefined, since cross-browser checking for undefined can be tedious.